warehouse/static/sass/blocks/_notification-bar.scss

```diff
@@ -39,6 +39,10 @@
   color: $white;
   padding: ($spacing-unit / 2);
 
+  .site-container {
+    position: relative;
```

Comment:
Given my earlier comment, this could be moved up to the notification-bar instead :)